packages/jest-runtime/src/index.ts

```diff
@@ -833,6 +844,31 @@ class Runtime {
     this._moduleMocker.clearAllMocks();
   }
 
+  // fileUrl.protocol cannot be set to a non-standard protocol, so we use string manipulation
+  private _createOutsideJestVmPath(path: string) {
+    return pathToFileURL(path)
+      .toString()
+      .replace(/^file:/, OUTSIDE_JEST_VM_PROTOCOL);
+  }
+  private _decodePossibleOutsideJestVmPath(
+    maybeUrl: string,
+  ): string | undefined {
+    let url: URL;
+    try {
+      url = new URL(maybeUrl);
+    } catch {
+      return undefined;
+    }
+    if (url.protocol !== OUTSIDE_JEST_VM_PROTOCOL) {
+      return undefined;
+    }
+    return fileURLToPath(
```

Comment:
`fileURLToPath` might not exist on all nodes

Reply:
I guess instead of the helper I'll just build the 'URL' string myself, it doesn't really need to be a proper URL with escaping and stuff, it's all for internal usage anyway

Reply:
sounds good - can add a todo to use it for for Jest 26? It was added in 10.12.0, I'm gonna set minimum version to at least 10.14.2 for some `vm.compileFunction` fixes

Reply:
Not sure if it actually helps though, because we can't manipulate the protocol property to something unofficial anyway, so there's always string manipulation involved actually